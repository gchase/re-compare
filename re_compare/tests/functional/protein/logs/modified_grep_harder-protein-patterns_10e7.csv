regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,58183,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,58183,"[-1, -1]"
DHKNLD.D,first_match,1,47894,"[613256, 613263]"
DHKNLD.D,consecutive,1,47894,"[613256, 613263]"
DHKNLD.D,consecutive,2,6,"[613396, 613403]"
DHKNLD.D,all_matches,2,47900,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,48598,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,48598,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,9,"[99267, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,48607,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,1,42585,"[214951, 214962]"
(GI)(G|R)P.Y..K..R,consecutive,1,42585,"[214951, 214962]"
(GI)(G|R)P.Y..K..R,consecutive,2,9,"[215382, 215393]"
(GI)(G|R)P.Y..K..R,all_matches,2,42594,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,1,73117,"[140635, 140641]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,1,73117,"[140635, 140641]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,2,16,"[140744, 140750]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,2,73133,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,1,51108,"[201599, 201602]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,1,51108,"[201599, 201602]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,2,215,"[226965, 226968]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,2,51323,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,1,43021,"[157516, 157521]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,1,43021,"[157516, 157521]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,2,8,"[157732, 157737]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,2,43029,"[-1, -1]"
P..L...KWE.C,first_match,0,59382,"[-1, -1]"
P..L...KWE.C,all_matches,0,59382,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,77487,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,77487,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,1,45148,"[811563, 811572]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,1,45148,"[811563, 811572]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,2,12,"[812073, 812082]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,2,45160,"[-1, -1]"
