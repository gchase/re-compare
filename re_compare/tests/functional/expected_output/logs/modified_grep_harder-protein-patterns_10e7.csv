regex,matchtype,matchnumber,time(ms),span
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),first_match,0,65389,"[-1, -1]"
(((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..)(F|Y),all_matches,0,65389,"[-1, -1]"
DHKNLD.D,first_match,1,48898,"[613256, 613263]"
DHKNLD.D,consecutive,1,48898,"[613256, 613263]"
DHKNLD.D,consecutive,2,4,"[613396, 613403]"
DHKNLD.D,all_matches,2,48902,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,first_match,1,47486,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,1,47486,"[98918, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,consecutive,2,16,"[99267, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DDP,all_matches,2,47502,"[-1, -1]"
(GI)(G|R)P.Y..K..R,first_match,1,48461,"[214951, 214962]"
(GI)(G|R)P.Y..K..R,consecutive,1,48461,"[214951, 214962]"
(GI)(G|R)P.Y..K..R,consecutive,2,9,"[215382, 215393]"
(GI)(G|R)P.Y..K..R,all_matches,2,48470,"[-1, -1]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),first_match,1,52600,"[140635, 140641]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,1,52600,"[140635, 140641]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),consecutive,2,13,"[140744, 140750]"
(((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N))(T|A|P|F|V),all_matches,2,52613,"[-1, -1]"
((N|S.TNH.Y)(F|W)N)(L|I),first_match,1,53358,"[201599, 201602]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,1,53358,"[201599, 201602]"
((N|S.TNH.Y)(F|W)N)(L|I),consecutive,2,177,"[226965, 226968]"
((N|S.TNH.Y)(F|W)N)(L|I),all_matches,2,53535,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,first_match,1,53186,"[157516, 157521]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,1,53186,"[157516, 157521]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,consecutive,2,10,"[157732, 157737]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A)T,all_matches,2,53196,"[-1, -1]"
P..L...KWE.C,first_match,0,78746,"[-1, -1]"
P..L...KWE.C,all_matches,0,78746,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),first_match,0,74734,"[-1, -1]"
((G|A.)(L|I|V|M)..HGTDT)(L|I|V|M),all_matches,0,74734,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,first_match,1,56059,"[811563, 811572]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,1,56059,"[811563, 811572]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,consecutive,2,14,"[812073, 812082]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y)S,all_matches,2,56073,"[-1, -1]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),first_match,1,46811,"[41350, 41358]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),consecutive,1,46811,"[41350, 41358]"
(((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W)R)(G|A|C|I|V|M),all_matches,1,250642,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),first_match,0,56659,"[-1, -1]"
(((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A))(L|I|V|M),all_matches,0,56659,"[-1, -1]"
(A|G....GK)(S|T),first_match,1,46231,"[56, 57]"
(A|G....GK)(S|T),consecutive,1,46231,"[56, 57]"
(A|G....GK)(S|T),consecutive,2,13,"[93, 94]"
(A|G....GK)(S|T),all_matches,2,46244,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)GT,first_match,1,91682,"[681320, 681325]"
((CP|.)(S|T)N)(I|L|V)GT,consecutive,1,91682,"[681320, 681325]"
((CP|.)(S|T)N)(I|L|V)GT,consecutive,2,2808,"[724975, 724981]"
((CP|.)(S|T)N)(I|L|V)GT,all_matches,2,94490,"[-1, -1]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,first_match,1,55359,"[413663, 413674]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,consecutive,1,55359,"[413663, 413674]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,consecutive,2,2706,"[1192335, 1192346]"
((((((P(L|I|V|M)..D)(G|A))(S|T))(A|C))(S|N))(G|A))(L|I|V|M|F|Y)Q,all_matches,2,58065,"[-1, -1]"
